src/Core/Adjectives/Adjective.fs

```diff
@@ -0,0 +1,12 @@
+﻿module Adjective
```

Comment:
Is this file related to parsing the word? Seems it does call adjective parser

Reply:
Well, it is just a container of all the stuff we need to know about adjectives, you can think of it as an Adjective class, just did not want to bring real classes here.

Reply:
Yes. Server uses only the `Storage` module and the exercise models (table entities).
But regretfully those entities are not POCOs yet, they depend on the `Core` stuff to set the props. That setting logic should be abstracted away and so eventually `Server` will not depend on the fat core but only on a small subset of it.

But for now I chunked away at least the validation part from "models".